### ISSUE TITLE
Include a heal all wounds button in heal wounds dialog

### ISSUE THREAD

**Issue by wake42** [sun-dragon-cult/fvtt-system-rqg#435]
Add a `Heal All` button to the hitlocation heal dialog
![image](https://user-images.githubusercontent.com/4853677/212615733-5414dae6-a5c8-420b-ad69-b9dc65182bf1.png)

It should not remove and `severed` status, but should remove `useless` - using the normal healing logic.

The button should be placed in the middle and not be the default button.

_Originally posted by @wake42 in https://github.com/sun-dragon-cult/fvtt-system-rqg/issues/423#issuecomment-1383563263_

**Comment by Moonpile** [comment]
I think this should be a list of the wounds with a text box for each saying how much to heal, then you click "Heal Wound".  There could be a third button that says "Heal All Wounds".

Imagine I have three wounds of 1, 2, and 3 hp, respectively.  Then I get a Heal 4.  I put 3 on the 3hp wound and the leftover one on the 2 hp wound, and click "Heal Wound".

See also the option to "leave a scar" in #439

**Comment by wake42** [comment]
I don't think you can cast a heal and have the "excess" points spill over into some other wound. 

The UX of how the extra "Heal All" button should be located maybe needs some thought though to make it obvious what does what, since the dialog will now do two different things, and only one of the buttons care about the wounds selected and points healed.

**Comment by wake42** [comment]
implemented in release 6.0.0
